apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-form-example/multi-input-form-example.component.html

```diff
@@ -1,8 +1,10 @@
 <form [formGroup]="customForm">
     <fieldset fd-fieldset>
         <div fd-form-item>
-            <label fd-form-label required="true">Valid Multi Input</label>
+            <label fd-form-label for="validMultiInput" required="true">Valid Multi Input</label>
             <fd-multi-input
+                inputId="validmultiInput"
```

Comment:
camel casing missed for the associated ids in this file, the `for` and `id` should have the same id

Reply:
sorry, missed it. fixed now.